python/mxnet/contrib/onnx/_import/op_translations.py

```diff
@@ -402,9 +438,9 @@ def max_pooling(attrs, inputs, cls):
                                                         'strides': 'stride',
                                                         'pads': 'pad',
                                                        })
+
     new_attrs = translation_utils._add_extra_attributes(new_attrs,
-                                                        {'pool_type': 'avg',
-                                                         'pooling_convention': 'valid'
+                                                        {'pooling_convention': 'valid'
                                                         })
     new_op = translation_utils._fix_pooling('max', inputs, new_attrs)
```

Comment:
if more than one operator needs the same fix during translation, we are putting that code in translation_utils so that there is no code repetition and redundancy. Here Max_Pooling and Avg_pooling are calling this method from translation_utils.

Reply:
Pooling is a same category for max, avg and so on. Will it help to unify them, you translator is getting too many detailed information about each individual operator, may be very hard for maintaining and making changes in future.

Reply:
We tried to keep the the translation logic for each operator discrete and separate, for sake of readability and extensibility. The previous design had clubbed the translation of various operators into a single class and a handful of methods, which resulted in the code becoming very convoluted with many if else blocks. Made it harder to add more operator support, debug existing operator implementations etc.. So we decided to change that design and have the translation logic for each operator separate. And if there are any repetitive tasks, then they can be performed in translation_utils. We can discuss this in-person too tomorrow morning.